smsdk/client.py

```diff
@@ -20,6 +21,7 @@
 import logging
 
 log = logging.getLogger(__name__)
+version_check_decorator = VersionCheckDecorator.version_check_decorator
```

Comment:
Should this be part of the `_version.py` file?

Reply:
Done.